website/content/en/docs/deployment/vanilla/guide-terraform.md

```diff
@@ -39,6 +52,18 @@ cluster_region="${CLUSTER_REGION}"
 EOF
 ```
 
+### Full Configuration
+
+A full list of inputs for the terraform stack can be found here and in the `variables.tf` file:
+
+| Name                 | Description                                       | Type   | Default  | Required |
```

Comment:
this will be hard to maintain, lets just link to the vars.tf

Reply:
done